src/course/syllabus/projects/TFB-design/schedule.md

```diff
@@ -36,29 +36,36 @@ schedule:
       start: 14:00
       end: 15:00
       type: project
-    - name: Technical spikes
+    - name: Project Planning
       start: 15:00
       end: 17:00
-      url: ../spikes
-    - name: Spike presentation prep
+      type: project
+      url: https://hackmd.io/@fac/SkyPKSwBw#/
+    - name: Code Planning + Choose Roles
```

Comment:
Let's split these up into two activities. We could cut Project Planning to 90 mins to give 15 to Choosing Roles and 15 more to Code Planning